### DISCUSSION TITLE
Extensible Admin

### DISCUSSION THREAD

**Discussion by olivermrbl** [medusajs/medusa#2830] [Feature Requests]
**What**
Allow developers to build custom components for the admin system in their local project or plugins

**Comment by oxneat** [comment]
Hello guys thank you for your efforts, it's a great job you are doing, so is there something new about this feature and how is the progress if you will

**Reply by oxneat** [reply]
@olivermrbl do you have an answer for this

**Reply by olivermrbl** [reply]
Have you seen [this RFC](https://github.com/medusajs/medusa/discussions/4116)?

If you have any feedback, we'd love to hear :)

**Comment by allcommerceco** [comment]
Is there a way to customize 
1) Adding a landing page before login
2) Branding in login

**Reply by alisafavi** [reply]
Hello
Did you find a way?

**Comment by ivanushkaPr** [comment]
We can extend our entitites in database, but cannot extend out create product admin UI.
So no one without programming skills can add new fields to product.

I find its kinda strange, that you need to impelent your own crud to do it and syncronize it with medusa backend.
